parama.html

```diff
@@ -27,14 +27,14 @@
           užpildytas automatiškai)</li>
         <li><em>Laukelis 3P</em> – Tavo pavardė (turėtų būti
           užpildytas automatiškai)</li>
-        <li><em>Laukelis 5</em> – 2016</li>
+        <li><em>Laukelis 5</em> – 2017</li>
         <li><em>Laukelis 6S</em> – Pažymėti varnele arba X</li>
         <li><em>Laukelis E1</em> – 2 (vienetas, turintis teisę gauti
           paramą)</li>
         <li><em>Laukelis E2</em> – 304138083</li>
         <li><em>Laukelis E3</em> – Kūrybinių technologijų</li>
         <li><em>Laukelis E4</em> – 2,00 (pagal savo norus)</li>
-        <li><em>Laukelis E5</em> – 2017</li>
+        <li><em>Laukelis E5</em> – 2018</li>
```

Comment:
o gal geriau rekomenduoti +5 metus, kad nereiktu kiekvienais metais nepamirst atnaujint?

Reply:
Gal, IMHO metai turetu likt del "teisiklingesnes" formos, in any case cia MR del duomenu tikslumo/teisingumo, changus galima daryti atskiru MR